packages/plugins/src/files/gridfs/gridfs-webhook.ts

```diff
@@ -77,8 +77,21 @@ export const gridfsHandler = async (
 
     if (req.method === 'GET') {
       const fileId = fileName;
+      const { s: signature, e: expiryTimestamp } = req.query;
 
       const file = await modules.gridfsFileUploads.getFileInfo(directoryName, fileId);
+      if (signature && expiryTimestamp) {
```

Comment:
Oh, security here: i can therefore just drop the query parameters and get infinite public access to that file? ;)